galley/testdatasets/validation/dataset/networking-v1alpha3-EnvoyFilter-invalid.yaml

```diff
@@ -3,5 +3,6 @@ kind: EnvoyFilter
 metadata:
   name: simple-envoy-filter
 spec:
-  workloadSelector:
-    labels:
+  configPatches:
+  - applyTo: LISTENER
+    patch: {}
```

Comment:
Is this always a must?

Reply:
the patch is. I just picked some random invalid config looking at our webhooks logic